src/main/java/com/everis/cars/boundary/CarResource.java

```diff
@@ -32,6 +41,13 @@
 	@Produces(MediaType.APPLICATION_JSON)
 	public List<Car> getAllCars();
 
+	/**
+	 * Create and retrieve the created car
+	 * 
+	 * @param car the car object to be created
```

Comment:
you can use @link to reference an object or class in javadocs

Reply:
I added the @link annotation to the comments to point the class commentend in javadocs.